client/wfnews-war/src/main/angular/src/app/components/wildfire-resources/wf-resources.component.ts

```diff
@@ -0,0 +1,102 @@
+import { Component } from '@angular/core';
+import { ResourcePanel } from '../../models/ResourcePanel';
+
+@Component({
+  selector: 'wf-resources',
+  templateUrl: './wf-resources.component.html',
+  styleUrls: [ './wf-resources.component.scss' ]
+})
+export class WildfirewResourcesComponent {
+  // This can be moved into a config, for easier changes later
+  public resourcePanels: ResourcePanel[] = [
+    {
+      title: 'Preparedness and Prevention',
+      resources: [
+        {
+          title: 'Wildfire Prevention', subtitle: 'BC Wildfire Services', url: 'https://www2.gov.bc.ca/gov/content/safety/wildfire-status/prevention',
```

Comment:
mock up's subtitle labeled as "BC Wildfires Service" .
To be consistent, should be "BC Wildfires Service", not plural ?

Reply:
Fixed